ql/src/test/queries/clientpositive/unionDistinct_3.q

```diff
@@ -3,6 +3,7 @@
 --! qt:dataset:src
 set hive.mapred.mode=nonstrict;
 set hive.explain.user=false;
+set hive.optimize.merge.adjacent.union.distinct=false;
```

Comment:
Why do we disable it here? I am worried that the result of non-EXPLAIN could change

Reply:
Now, I understand this is intended